### ISSUE TITLE
ProTips

### ISSUE THREAD

**Issue by Shtoyan** [davidoskky/SplitTimer#5]
https://github.com/davidoskky/SplitTimer/commit/6f5d005bb904a196c5049c106aa063bb31747d43#diff-174340207fba6f0897f9646173874cf380a1c1a824fd85925aa6030eff4d839eR10

1. Just a note. No need to set variables to false / 0. They are Always false on init., unless you alter it in defaultproperties / config / replication, etc.

https://github.com/davidoskky/SplitTimer/commit/6f5d005bb904a196c5049c106aa063bb31747d43#diff-174340207fba6f0897f9646173874cf380a1c1a824fd85925aa6030eff4d839eR24

2. You can use `bAddToServerPackages` flag instead. And remove `PreBeginPlay()` completely.

3. Stop checking game states in `Tick()`, add a Role check for dedicated servers and disable tick for them.

**Comment by davidoskky** [comment]
Thanks, that is great news!
It will simplify the code a bit, which is always nice...

btw do you know if an editor exists that does linting and syntax checks?

**Comment by Shtoyan** [comment]
TLDR: i'll make a PR.

> btw do you know if an editor exists that does linting and syntax checks?

VSCode + https://marketplace.visualstudio.com/items?itemName=EliotVU.uc

**Comment by davidoskky** [comment]
Wow, thanks a lot!
